odl/solvers/util/callback.py

```diff
@@ -483,7 +492,17 @@ def __init__(self, *args, **kwargs):
         self.fig = kwargs.pop('fig', None)
         self.step = kwargs.pop('step', 1)
         self.saveto = kwargs.pop('saveto', None)
+        try:
+            self.saveto_formatter = self.saveto.format
+        except AttributeError:
+            self.saveto_formatter = self.saveto
```

Comment:
Verified to work with 0.18.1 but failing with 0.19, all other packages kept constant.

Not quite sure how to handle this on our side.